src/web/app/pages-student/student-course-details-page/student-course-details-page.component.spec.ts

```diff
@@ -36,48 +37,55 @@ describe('StudentCourseDetailsPageComponent', () => {
   });
 
   it('should snap with populated fields', () => {
-    const courseAttributes: any = {
-      createdAt: '2019-03-12T09:13:29.272Z',
-      id: 'CS3281',
-      name: 'Thematic Systems',
-      timeZone: 'Asia/Singapore',
+    const student: Student = {
+      courseId: '1.1.c-demo2',
+      email: '1@1.com',
+      lastName: '1',
+      name: '1',
+      comments: '',
+      requestId: '5cb7d3320000b3b0cb4e5208',
+      joinState: JoinState.NOT_JOINED,
+      sectionName: 'Tutorial Group 2',
+      teamName: 'Team 2',
     };
-    const instructorDetails: any = [
-      {
-        name: 'Ayush',
-        email: 'ayush@ayush.com',
-      }, {
-        name: 'Byush',
-        email: 'byush@byush.com',
-      },
-    ];
-    const studentDetails: any = {
-      email: 'cyush@cyush.com',
-      course: 'CS3281',
-      name: 'Cyush',
-      googleId: 'cyush@gmail.com',
-      team: 'CSquad',
-      section: 'Section 1',
+
+    const instructorDetails: Instructor[] = [{
+      courseId: '1.1.c-demo2',
+      displayedToStudentsAs: 'Instructor',
+      isDisplayedToStudents: true,
+      email: '1@1.com',
+      name: '1',
+      joinState: JoinState.JOINED,
+    }];
+
+    const course: Course = {
+      courseId: '1.1.c-demo2',
+      courseName: 'Sample Course 101',
+      creationDate: '13 Mar 2019',
+      deletionDate: 'Not Applicable',
+      requestId: '5cb7d33200015f90f1f836cb',
+      timeZone: 'UTC',
     };
-    const teammateProfiles: any = [
-      {
-        shortName: 'ash',
-        email: 'ash@ash.com',
-        institute: 'NUS',
-        nationality: 'Indian',
-        gender: Gender.MALE,
-        pictureKey: 'something.jpg',
-      },
+
+    const teammateProfiles: StudentProfileWithPicture[] = [
       {
-        shortName: 'bsh',
-        email: 'bsh@bsh.com',
-        gender: Gender.OTHER,
-        pictureKey: 'something2.jpg',
+        photoUrl: '/assets/images/profile_picture_default.png',
+        studentProfile: {
+          email: 'iam2@hello.com',
+          gender: Gender.MALE,
+          institute: 'nus',
+          moreInfo: 'Misc',
+          name: '2',
+          nationality: 'Andorran',
+          pictureKey: '',
```

Comment:
You should remove `pictureKey` in `StudentProfileData`?

Reply:
Removed and updated!